internal/impl/wasm/build.sh

```diff
@@ -1,2 +1,3 @@
 #!/bin/sh
-tinygo build -scheduler=none -target=wasi -o uppercase.wasm ../../../public/wasm/examples/tinygo
+#TODO
+#tinygo build -scheduler=none -target=wasi -o uppercase.wasm ../../../public/wasm/examples/tinygo
```

Comment:
It is actually! We don't do this in Redpanda because wasip1 requires too much memory, but there are no restrictions like that here. I will not switch the old wasm processor in this PR.